src/main.rs

```diff
@@ -26,7 +26,7 @@ mod sources;
 /// Checking what changed about dependencies between versions.
 #[derive(Debug, StructOpt)]
 struct Opts {
-    /// Git range specifying the changes to inspect.
+    #[structopt(short = "r", long = "revspec")]
```

Comment:
Why do you add this? I did intend to have this as a positional argument, without any `--flag`.